### DISCUSSION TITLE
Giving credit to enarx when copying files to other Apache 2.0 projects?

### DISCUSSION THREAD

**Discussion by waywardgeek** [enarx/enarx#2538] [Q&A] [answered]
I have been asked to add SEV-SNP support to Cloud Hypervisor, and I see Enarx already has SEV-SNP support.  I assume since you folks licensed it under Apache 2.0, you're OK with me copying files and portions of files according to the license.  One thing I noticed was that your files say "// SPDX-License-Identifier: Apache-2.0", and does not list Enarx copyright owners by name.

If I do copy Enarx files to other Apache 2.0 projects, specifically Cloud Hypervisor, would you like me to change the header to a traditional copyright header that specifically names you folks?

**Comment by nickvidal** [comment]
Hi @waywardgeek, yes, you are free to copy the code and give the appropriate attribution as suggested. Thanks!
